cmd/clusterctl/client/cluster/mover.go

```diff
@@ -200,8 +210,10 @@ func (o *objectMover) move(graph *objectGraph, toProxy Proxy) error {
 
 	// Ensure all the expected target namespaces are in place before creating objects.
 	log.V(1).Info("Creating target namespaces, if missing")
-	if err := o.ensureNamespaces(graph, toProxy); err != nil {
-		return err
+	if !o.isDryrun {
+		if err := o.ensureNamespaces(graph, toProxy); err != nil {
```

Comment:
When dry running, I think that we should also avoid to pause/unpause the clusters, wdyt?

Reply:
yes, will add it